anvil-rstudio-base/README.md

```diff
@@ -1,19 +1,76 @@
+
+
+### Applications
+
+The AnVIL R Docker contains the [RStudio Server](https://www.rstudio.com/products/rstudio-server/) application which supports a web browser version of RStudio. 
+
+### Libraries
+
+The following packages are pre-loaded in the AnVIL RStudio image:
+
+* [tidyverse](https://www.tidyverse.org/packages/): a suite of packages designed for data sciences 
+* [google-cloud-sdk](https://cloud.google.com/sdk/): tools for the google cloud platform
+* [googleCloudStorageR](http://code.markedmondson.me/googleCloudStorageR/): an R library for interacting with google cloud storage
+* [bigrquery](https://github.com/r-dbi/bigrquery): a package to allow interaction with Google's BigQuery
+* [DataBiosphere/Ronaldo](https://github.com/DataBiosphere/Ronaldo): a package of functions commonly used for R Notebooks created from Leonardo
+
+### Customizations
+
+The AnVIL RStudio image is customized to disable the RStudio login screen. Future customizations will include:
+
+* RStudio hooks to refresh credentials 
+* An RStudio pause after inactivity
+
+## Using RStudio in Terra
+
+Detailed instructions on accessing the AnVIL RStudio image can be found in the "[Working with project-specific images](https://broadinstitute.zendesk.com/knowledge/articles/360037269472)" documentation in Terra support.
+
+## Saving RStudio Data
+
+Code and data generated in Terra RStudio will not be saved. Below are three suggestions for saving your work.
+
+### 1. Copying RStudio work to a workspace google bucket
+
+Use the [gsutil](https://cloud.google.com/storage/docs/gsutil) tool to copy files to your workspace google bucket. The "[Working with project-specific images](https://broadinstitute.zendesk.com/knowledge/articles/360037269472)" documentation in Terra support demonstrates how to identify the url for a workspace google bucket. After identifying the workspace google bucket url, you can copy files to a google bucket using the bash command: 
+
+    !gsutil cp ./* gs://GoogleBucketURL
+
+### 2. Downloading RStudio files to your local computer
+Once your files are copied to workspace google bucket, you can access them by selecting the Data tab of the workspace and choosing the Files option on bottom left. This will display the files available in your google bucket. By selecting a file, you can download it directly. Additionally, [this Terra support document](https://support.terra.bio/hc/en-us/articles/360029251091-Broad-Genomics-Downloading-data-from-a-Terra-workspace) details alternative techniques you can use to download data files. 
+
+### 3. Checking code into GitHub
+You can install Git on RStudio and use it to [check code into GitHub](https://help.github.com/en/github/importing-your-projects-to-github). 
+
+## Versioning
+
+All updates and changes to the current Docker image are listed in the Rstudio image [CHANGELOG.md file](CHANGELOG.md). 
```

Comment:
move this somewhere else, maybe in summary? For `Versioning`, it's usually about how we version the image

Reply:
Changed to "RStudio Docker Updates"